### ISSUE TITLE
bug: core object marker: icons have wrong text for screen reader

### ISSUE THREAD

**Issue by Seamoo13** [SAP/fundamental-ngx#3557]
#### Is this a bug, enhancement, or feature request?

bug

#### Briefly describe your proposal.

When you use a screen reader on the object marker with icons only section, a large body of text, which does not give the listener a clear understanding of what is there, is read. What should be read is the aria-label 
env: https://fundamental-ngx.netlify.app/#/core/object-marker

#### If this is a bug, please provide steps for reproducing it.

1. enable a screen reader
2 click on an icon in the object marker icons only section
3. listen/read output

**actual:** large body of text read
![Screenshot_1](https://user-images.githubusercontent.com/47522152/95178704-6ab48080-07c8-11eb-9b7f-6d602d631964.jpg)

**Expected:** read aria-labels
![Screenshot_4](https://user-images.githubusercontent.com/47522152/95178709-6c7e4400-07c8-11eb-894d-c76c118a03b2.jpg)

**Comment by Lokanathan-k** [comment]
closed as part of this PR https://github.com/SAP/fundamental-ngx/pull/4081
